physics/CONV/progsigma_calc.f90

```diff
@@ -186,11 +185,11 @@ subroutine progsigma_calc (im,km,flag_init,flag_restart,flag_shallow,&
                sigmab(i)=(ZZ*(termA(i)+cvg))/(DEN+(1.0-ZZ))
                if(sigmab(i)>0.)then
                   sigmab(i)=MIN(sigmab(i),0.95)  
-                  sigmab(i)=MAX(sigmab(i),0.01)
+                  sigmab(i)=MAX(sigmab(i),sigmind_new)
                endif
             endif!cnvflg
          enddo
-      endif
+!      endif
```

Comment:
One comment line that was missed when other comment lines were removed.

Reply:
good catch! line removed